src/screens/Home/Home.js

```diff
@@ -71,27 +72,53 @@ function Home() {
   const accountTotalBalances = useRootSelector(accountTotalBalancesSelector);
   const accountCollectibleCounts = useAccountCollectibleCounts();
   const user = useUser();
+  const wallet = useRootSelector((root) => root.wallet.data);
+  const accountAddress = useRootSelector(activeAccountAddressSelector);
+
   const dispatch = useDispatch();
+
+  React.useEffect(() => {
+    if (!wallet) {
+      getSupportedBiometryType((biometryType) => {
+        if (biometryType) {
+          Modal.open(() => <BiometricModal biometricType={biometryType} />);
```

Comment:
Could we delay this by maybe 2 seconds? This shows even before home has fully rendered and results in quite a sharp-ended UX...

Reply:
added the loader also till wallet is created

https://user-images.githubusercontent.com/66996635/148932911-5f5092be-49b1-4178-9c6c-d4084554237f.mov

Reply:
That's great thanks - i've just noticed however that three UI elements there popped over each other. We've got the backup notification, register your ENS and i can see one more, do you know what that is?

Reply:
yes, it is "You can switch accounts by tapping here", It is due to recent changes of key based wallet integration code.

Reply:
https://user-images.githubusercontent.com/66996635/149080112-8526c7c3-0a94-45f0-9727-c6f131391f91.mov